workflow/rocoto/gfs_tasks.py

```diff
@@ -591,7 +592,10 @@ def prepsnowobs(self):
     def snowanl(self):
 
         deps = []
-        dep_dict = {'type': 'task', 'name': f'{self.run}prepsnowobs'}
+        if f'@H' == '00':
```

Comment:
`@H` is a rocoto thing, so this doesn't have any meaning to python when this function is run and this won't work as-is. #2928 is adding support for the undocumented rocoto dependency `<taskvalid>`. Once that is in, we can use it to make a dependency that works here:

```
<or>
    <and>
        <not><taskvalid task="{self.run}prepsnowcover"/></not>
        <taskdep task="{self.run}prep"/>
    </and>
    <taskdep task="{self.run}prepsnowcover"/>
</or>
```

Reply:
Thanks @WalterKolczynski-NOAA for the information. Your introduced `<taskvalid>` above is exactly what we want here. For now, I made some changes here for a place holder. 
```
                <or>
                        <taskdep task="gdasprep"/>
                        <taskdep task="gdasprepsnowcover"/>
                </or>

```

Reply:
@WalterKolczynski-NOAA Do you want me to wait for your PR https://github.com/NOAA-EMC/global-workflow/pull/2928 or proceed to review this PR? I can update to use `<taskvalid>` after your PR is merged. Please let me know. Thanks.